layouts/partials/ecosystem/registry/entry.html

```diff
@@ -187,7 +201,17 @@ <h6><i class="fa-solid fa-forward"></i> Quick Install</h6>
                 <div class="fw-bold">
                     {{ . }}
                 </div>
-                <small>License</small>
+                {{ if eq . "Commercial" }}
+                <small data-bs-toggle="tooltip" 
+                    title="Commercial License: This project has a commercial license. End users may need to check with the provider for specific usage rights and restrictions. For more details on common license types, visit the Open Source Initiative at opensource.org/licenses.">
+                    <a href="https://opensource.org/licenses" target="_blank" class="text-muted">License</a>
+                </small>
+            {{ else }}
+                <small data-bs-toggle="tooltip" 
+                    title="Open-Source License: This project is distributed under an open-source license. End users are encouraged to review the license terms to ensure compatibility with their usage. Visit the Open Source Initiative for more details.">
+                    <a href="https://opensource.org/licenses" target="_blank" class="text-muted">License</a>
+                </small>
+            {{ end }}
```

Comment:
```suggestion
                {{ if eq . "Commercial" }}
                <small data-bs-toggle="tooltip" 
                    title="Commercial License: This project has a commercial license. End users may need to check with the provider for specific usage rights and restrictions.">
                    License
                </small>
            {{ else }}
                <small data-bs-toggle="tooltip" 
                    title="Open-Source License: This project is distributed under an open-source license. End users are encouraged to review the license terms to ensure compatibility with their usage.">
                    License
                </small>
            {{ end }}
```

Let's remove the links for now, we can improve on that in a later iteration